qwandaq/src/main/java/life/genny/qwandaq/utils/FilterUtils.java

```diff
@@ -144,7 +148,7 @@ public Ask getAddFilterGroupBySearchBE(String questionCode) {
         BaseEntity source = beUtils.getBaseEntity(sourceCode);
         BaseEntity target = beUtils.getBaseEntity(sourceCode);
 
-        Ask ask = qwandaUtils.generateAskFromQuestionCode(Question.QUE_ADD_FILTER_SBE_GRP, source, target);
+        Ask ask = qwandaUtils.generateAskFromQuestionCode(Question.QUE_ADD_FILTER_SBE_GRP, source, target, new CapabilitySet(target), new ReqConfig());
```

Comment:
Need to use `CapabilitiesManager.getUserCapabilities()` or `Capabilities.getEntityCapabilities(user)` to fetch user `CapabilitySet`